dds_glossary/main.py

```diff
@@ -1,194 +1,48 @@
+        dsn=SENTRY_DSN,
+        # Set traces_sample_rate to 1.0 to capture 100%
+        # of transactions for performance monitoring.
+        traces_sample_rate=1.0,
+        # Set profiles_sample_rate to 1.0 to profile 100%
+        # of sampled transactions.
+        # We recommend adjusting this value in production.
+        profiles_sample_rate=1.0,
     )
+    app = FastAPI()
+    app.state.controller = GlossaryController()
+    app.state.templates = Jinja2Templates(directory="templates")
+    app.include_router(router)
+    return app
 
 
-@app.post("/init_datasets")
-def init_datasets(
-    _api_key: dict = Depends(get_api_key),
-    reload: bool = False,
-) -> JSONResponse:
-    """Initialize the datasets.
-
-    Args:
-        _api_key (dict): The API key.
-        reload (bool): Flag to reload the datasets. Defaults to False.
-
-    Returns:
-        JSONResponse: The status of the init request.
-    """
-    saved_datasets, failed_datasets = controller.init_datasets(reload=reload)
-    return JSONResponse(
-        content={
-            "saved_datasets": saved_datasets,
-            "failed_datasets": failed_datasets,
-        },
-        media_type="application/json",
-        status_code=HTTPStatus.OK,
-    )
-
-
-@app.get("/schemes")
-def get_concept_schemes(lang: str = "en") -> JSONResponse:
-    """
-    Returns all the saved concept schemes.
-
-    Args:
-        lang (str): The language. Defaults to "en".
-
-    Returns:
-        JSONResponse: The concept schemes.
-    """
-    return JSONResponse(
-        content={
-            "concept_schemes": controller.get_concept_schemes(lang=lang),
-        },
-        media_type="application/json",
-        status_code=HTTPStatus.OK,
-    )
-
-
-@app.get("/concepts")
-def get_concepts(concept_scheme_iri: str, lang: str = "en") -> JSONResponse:
-    """
-    Returns all the saved concepts for a concept scheme.
-
-    Args:
-        concept_scheme_iri (str): The concept scheme IRI.
-        lang (str): The language. Defaults to "en".
-
-    Returns:
-        JSONResponse: The concepts.
-    """
-    return JSONResponse(
-        content={
-            "concepts": controller.get_concepts(concept_scheme_iri, lang=lang),
-        },
-        media_type="application/json",
-        status_code=HTTPStatus.OK,
-    )
-
-
-@app.get("/concept")
-def get_concept(concept_iri: str, lang: str = "en") -> JSONResponse:
-    """
-    Returns a concept.
-
-    Args:
-        concept_iri (str): The concept IRI.
-        lang (str): The language. Defaults to "en".
-
-    Returns:
-        JSONResponse: The concept.
-    """
-    return JSONResponse(
-        content=controller.get_concept(concept_iri, lang=lang),
-        media_type="application/json",
-        status_code=HTTPStatus.OK,
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="DDS Glossary")
+    parser.add_argument(
+        "-r", dest="f_reload", action="store_true", help="enable auto-reloading"
```

Comment:
@jsvgoncalves should set default value to false.

Reply:
This is the default, see argparse docs:

> `store_true` and `store_false` - These are special cases of `store_const` used for storing the values True and False respectively. In addition, they create default values of `False` and `True` respectively. 


https://docs.python.org/3/library/argparse.html#action